src/arkhamdb/DeckImporterMain.ttslua

```diff
@@ -245,11 +248,20 @@ end
 ---@param slots: The slot list for cards in this deck. Table key is the cardId, value is the number
 --     of those cards which will be spawned
 function maybeAddSummonedServitor(slots)
-  for cardId, cardCount in pairs(slots) do
-    -- spawn additional minicard for 'Summoned Servitor'
-    if cardId == "09080" then
-      slots["09080-m"] = 1
-      return
+  if slots["09080"] ~= nil then
+    slots["09080-m"] = 1
+  end
+end
+
+-- On the Mend should have 1-per-investigator copies set aside, but ArkhamDB always sends 1.  Update
+-- the count based on the investigator count
+---@param slots: The slot list for cards in this deck. Table key is the cardId, value is the number
+--     of those cards which will be spawned
+function maybeAddOnTheMend(slots)
+  if slots["09006"] ~= nil then
+    local playArea = getObjectFromGUID(PLAY_AREA_GUID)
+    if playArea ~= nil then
+      slots["09006"] = playArea.call("getInvestigatorCount")
```

Comment:
Wouldn't it be better to directly access the playercounter instead of doing the additional step via the playarea?

Reply:
This is a design choice - depends on whether we consider the investigator count as a separate element, or part of the play area.  For now it's part of the area so I'm leaving it there and using the play area as a parent object/interface.  Possibly more changes coming on this, but again trying to be minimally disruptive to make the fix in the freeze window.